packages/chakra-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -0,0 +1,146 @@
+import { Select } from '@chakra-ui/react'
+import React from 'react'
+import PropTypes from 'prop-types'
+import { WidgetProps, utils } from '@rjsf/core'
+
+const nums = new Set(['number', 'integer'])
+const { guessType, asNumber } = utils;
+
+/**
```

Comment:
Select and radio descriptions are not rendering (see the "Widgets" tab on the playground):

![image](https://user-images.githubusercontent.com/1689183/111927036-96c97280-8a85-11eb-8a50-2375ea253b51.png)


Default theme:

![image](https://user-images.githubusercontent.com/1689183/111927045-9cbf5380-8a85-11eb-9adc-89eb0d47e5fc.png)